apps/api/src/domain/services/query-services/media-management/fs-media-prober.ts

```diff
@@ -2,11 +2,59 @@ import { isNonNegativeFiniteNumber } from '@coscrad/validation-constraints';
 import * as ffmpeg from 'fluent-ffmpeg';
 import { promisify } from 'util';
 import { Maybe } from '../../../../lib/types/maybe';
-import { NotFound } from '../../../../lib/types/not-found';
-import { isNullOrUndefined } from '../../../utilities/validation/is-null-or-undefined';
+import { NotFound, isNotFound } from '../../../../lib/types/not-found';
 import { IMediaProber, RawMediaInfo } from './media-prober.interface';
 
-const ffprobe = promisify(ffmpeg.ffprobe);
+/**
+ * TODO Consider making this a nested entity with type decorators and behaviour.
+ * If so, leverage this in `Photograph` class.
+ */
+export type PhotographDimensions = {
```

Comment:
Actually, there are already 2 such entities, and we need to consolidate the 3 definitions.

Reply:
done